### ISSUE TITLE
TKW assurance: positive test for Nullification message, the FHIR patient resource does not output the patient NHS number

### ISSUE THREAD

**Issue by Kusnaditjung** [nhsconnect/integration-adaptor-111#152]
**Description**
For the TKW assurance test for the Nullification message, the response is successful, but after the adapter translates the message, the FHIR Patient resource output does not have an NHS number, while the original nullification CDA has the NHS number. Normally for CDA, it comes either just an NHS number or full demographics, and in this case, it is just an NHS number. The patient NHS number already has a mapping, and it was output correctly in other messages, but not in the nullification messages, so I think this is just a coding error and a simple fix, but this is very important for the validity of the FHIR output.   
 
Here is the original element of the patient NHS number for CDA nullification.
```xml
<recordTarget typeCode="RCT" contextControlCode="OP">
		<npfitlc:contentId root="2.16.840.1.113883.2.1.3.2.4.18.16" extension="COCD_TP145020UK03#PatientRole"/>
		<patientRole classCode="PAT">
			<templateId root="2.16.840.1.113883.2.1.3.2.4.18.2" extension="COCD_TP145020UK03#PatientRole"/>
			<id root="2.16.840.1.113883.2.1.4.1" extension="9900002558"/> <--- This is the nhs number
		</patientRole>
	</recordTarget>
```
and here is the output of the FHIR Patient resource for the adapter at the moment.

```json
{
      "fullUrl": "urn:uuid:5ce0adc2-be17-4f62-9d2c-b43cbfb2ad23",
      "resource": {
        "resourceType": "Patient"
      }
},
```

**Component**
NHS111 adapter

**Version**
0.5.0

**Step to Reproduce**
- make sure the adapter is running, with PEM111_SOAP_SEND_TO=[nigx endpoint]/report
- copy **Nullification_POCD_EX130001UK05_01.xml** file template from **TKW/contrib/ITK_2_01_Test_Messages/111/Negative_111_Templates**
  to **\TKW\config\ITK_111\TestRun\Testfiles**
- run 1.Clear_Folders.bat
- run 3.Transmit_TestFiles.bat
- check the adapter queue, check on FHIR patient resource

**Test Data**
from TKW assurance 

**Preconditions**
None

**Environment**
local environment

**Severity**
Severe

**Priority**
Urgent

**Expected Behavior**

the FHIR Patient resource should be like this:
```json
{
      "fullUrl": "urn:uuid:b31ceae5-47be-441a-8b86-8b59a46f4fd7",
      "resource": {
        "resourceType": "Patient",
        "identifier": [
          {
            "extension": [
              {
                "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1",
                "valueCodeableConcept": {
                  "coding": [
                    {
                      "system": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1",
                      "code": "02"
                    }
                  ]
                }
              }
            ],
            "system": "https://fhir.nhs.uk/Id/nhs-number",
            "value": "993254128"
          }
        ],
        "active": true       
      }
    }
```

**Comment by dawidskainos** [comment]
Will be fixed in 0.5.1 release

**Comment by Kusnaditjung** [comment]
This can be closed now. Fix in 0.5.1
